apps/financial-aid/web-osk/src/routes/application/ContactInfo/contactInfo.tsx

```diff
@@ -14,24 +14,25 @@ import {
   isEmailValid,
 } from '@island.is/financial-aid/shared/lib'
 
-const EmailForm = () => {
+const ContactInfo = () => {
   const router = useRouter()
 
   const { form, updateForm } = useContext(FormContext)
-  const [hasError, setHasError] = useState(false)
+  const [hasEmailError, setHasEmailError] = useState(false)
+  const [hasPhoneNumberError, setHasPhoneNumberError] = useState(false)
 
   const navigation: NavigationProps = useFormNavigation(
     router.pathname,
   ) as NavigationProps
 
   const errorCheck = () => {
-    if (form?.emailAddress === undefined) {
-      setHasError(true)
+    if (form?.emailAddress === undefined || !isEmailValid(form?.emailAddress)) {
+      setHasEmailError(true)
       return
     }
 
-    if (!isEmailValid(form?.emailAddress)) {
-      setHasError(true)
+    if (form?.phoneNumber === undefined || form?.phoneNumber.length !== 7) {
```

Comment:
is it legit to check if number has 7 in length, what if I have a phone number from abroad? ✈️ 🛄